src/index.js

```diff
@@ -16,21 +17,46 @@ const {
   APPROVE_ONLY,
   API_URL,
   TARGET,
+  PR_NUMBER,
 } = getInputs()
 
 const GITHUB_APP_URL = 'https://github.com/apps/dependabot-merge-action'
 
 async function run() {
   try {
-    const { pull_request: pr } = github.context.payload
+    const { pull_request, workflow } = github.context.payload
 
-    if (!pr) {
+    const isPullRequest = Boolean(pull_request)
+    const isWorkflowDispatch = Boolean(workflow)
```

Comment:
these boolean variables are unnecessary. since you're checking for their truthyness you don't need temp variables